### PR TITLE
few changes in connected cup to remove the dependent with the configu…

### PR DESCRIPTION
**Pull request by ayyoob**
…ration file

### CODE REVIEW

File: modules/samples/connectedcup/component/plugin/src/main/java/org/coffeeking/connectedcup/plugin/exception/ConnectedCupDeviceMgtPluginException.java [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.coffeeking.connectedcup.plugin.exception;
+
+
+public class ConnectedCupDeviceMgtPluginException extends Exception{
+
+    private String errorMessage;
```
Comment (prabathabey) [inline modules/samples/connectedcup/component/plugin/src/main/java/org/coffeeking/connectedcup/plugin/exception/ConnectedCupDeviceMgtPluginException.java:23]: Having the error message stored in a separate variable is redundant. Might be a good idea to remove it.

File: modules/samples/connectedcup/component/plugin/src/main/java/org/coffeeking/connectedcup/plugin/impl/dao/util/ConnectedCupUtils.java [hunk 2]
```diff
@@ -30,15 +36,52 @@
 
     private static Log log = LogFactory.getLog(ConnectedCupUtils.class);
 
-    public static String getDeviceProperty(Map<String, String> deviceProperties, String property) {
+    public static String getDeviceProperty(List<Device.Property> deviceProperties, String propertyKey) {
+        String deviceProperty = "";
+        for(Device.Property property :deviceProperties){
+            if(propertyKey.equals(property.getName())){
```
Comment (prabathabey) [inline modules/samples/connectedcup/component/plugin/src/main/java/org/coffeeking/connectedcup/plugin/impl/dao/util/ConnectedCupUtils.java:42]: Fix formatting issues in this particular line and the one that follows it.

File: modules/samples/connectedcup/component/plugin/src/main/java/org/coffeeking/connectedcup/plugin/exception/ConnectedCupDeviceMgtPluginException.java (hunk 1 above)
Comment (ayyoob) [inline modules/samples/connectedcup/component/plugin/src/main/java/org/coffeeking/connectedcup/plugin/exception/ConnectedCupDeviceMgtPluginException.java:23]: Fixed with https://github.com/wso2/product-iots/pull/181/commits/dca55c73852e621175bcf44800ee9b19cd95178f

File: modules/samples/connectedcup/component/plugin/src/main/java/org/coffeeking/connectedcup/plugin/impl/dao/util/ConnectedCupUtils.java (hunk 2 above)
Comment (ayyoob) [inline modules/samples/connectedcup/component/plugin/src/main/java/org/coffeeking/connectedcup/plugin/impl/dao/util/ConnectedCupUtils.java:42]: Fixed with https://github.com/wso2/product-iots/pull/181/commits/dca55c73852e621175bcf44800ee9b19cd95178f

**Pull request merged**